packages/devtools_app/lib/src/flutter/common_widgets.dart

```diff
@@ -130,3 +131,12 @@ TextStyle primaryColorLight(TextStyle style, BuildContext context) {
     fontWeight: FontWeight.w300,
   );
 }
+
+/// Generates an [ErrorReporter] that shows a [SnackBar].
+ErrorReporter showErrorSnackBar(BuildContext context) {
```

Comment:
this is exactly the sort of functionality that is good to add to common_widgets.